### ISSUE TITLE
Non-string values as environment parameters.

### ISSUE THREAD

**Issue by timonag** [allure-framework/allure-mocha#30]
Starting from 1.3.1 `addEnvironment` method no longer accepts any types except String as a second param.
For example, webdriver library I'm using returns an array with single item during this call `driver.getCapabilities().getPlatform().getPartOfOsName() // ['linux']` and this causes allure to fail without a stack trace and no report is created:
```
allure.addEnvironment("operating system platform", driver.getCapabilities().getPlatform().getPartOfOsName())
```
Rolling back to 1.3.0 works. Explicitly converting an array to string works on both versions.

I believe it's a minor issue belonging to `allure-js-commons`, but I could not reproduce the bug with its tests.

**Comment by just-boris** [comment]
I guess that I know what caused the problem.

I have updated [js2xml](https://github.com/michaelkourlas/node-js2xmlparser) dependency version. Seems like new version has behavior changes.

What output do you to expect in your particular use-case?

**Comment by timonag** [comment]
Converting values to string would work for me.
But I believe that **js2xml** devs had a reason to turn off type conversion when writing xml attributes, so as for me unless other issues arise It's not worth an effort.

**Comment by just-boris** [comment]
Not a solution for your issue, but I have prepared a PR #31 that gives better logging in such case.

Hope it will help to find wrong places in your code easier.

**Comment by just-boris** [comment]
I have merged the PR, and released new version `1.3.2`, I hope it will help to find such errors.

Regarding the initial bug report, we have decided, that new behavior is a good change and it is fine to throw an error when you are providing the invalid environment. Now you will get an explicit error message about it and you will be able to fix it easily.
